README.md

```diff
@@ -10,4 +10,239 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+TODO: Jeremiah
+
+### Insurance Card
+
+#### `GET /api/insuranceCards?owner=USERNAME` - Get insurance cards by owner
```

Comment:
urls are generally case-insensitive, so we shouldn't use camel case like this -- I would go for `/api/insurance-cards` or just `/api/cards` instead